core.osgi/src/main/java/com/ericsson/bss/cassandra/ecchronos/core/osgi/commands/PrintUtils.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2019 Telefonaktiebolaget LM Ericsson
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.ericsson.bss.cassandra.ecchronos.core.osgi.commands;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Locale;
+
+public final class PrintUtils
+{
+    static final SimpleDateFormat DATE_FORMATTER = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss", Locale.US);
```

Comment:
Ok, I was using SimpleDateForatter since it was used somewhere else in the code (RepairStateImpl), but there is was used inside a ThreadLocal...
I guess there will be no threading issues when executing this code from a Command :-)
But I'll change it anyways!